src/rpm/builder.rs

```diff
@@ -196,18 +157,20 @@ impl RPMBuilder {
         self
     }
 
-    /// Override the automatically-detected name of the build host with a new value
-    pub fn build_host(mut self, build_host: &str) -> Self {
-        self.build_host = build_host.to_owned();
+    pub fn build_host(mut self, build_host: impl AsRef<str>) -> Self {
```

Comment:
Could also you add docstrings to `build_host`, `build_time`, and `add_changelog` entry with your examples